### PR TITLE
Feature/SOF-7326 feat: add tests for JL session

### PR DESCRIPTION
**Pull request by VsevolodX**
Current implementation works if chrome security is reduced to allow Cypress to look into iframe with foreign URL than the host page

### CODE REVIEW

File: tests/cypress/support/widgets/JupyterLiteSession.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+export default class JupyterLiteSession extends Widget {
+    wrappedSelectors: typeof selectors;
+
+    constructor() {
+        super(selectors.wrapper);
+        this.wrappedSelectors = this.getWrappedSelectors(selectors);
+    }
+
+    waitForVisible() {
+        return cy
```
Comment (k0stik) [inline tests/cypress/support/widgets/JupyterLiteSession.ts]: It should something like `this.browser.iframe(selectors.wrapper).find(selectors.main)`
Check web-app for examples

Comment (VsevolodX) [reply tests/cypress/support/widgets/JupyterLiteSession.ts]: The problem with that is basically we would need to reimplement all of the methods available in browser for iframeBrowser. Don't quite understand where's the advantage over using `cy` commands that are already fully available and well-documented. 
This iframeBrowser has `#body` as a base for commands, while during chaining I need to apply method to the result, not the`#body`.

File: tests/cypress/support/step_definitions/I click on link.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { When } from "@badeball/cypress-cucumber-preprocessor";
```
Comment (timurbazhirov) [inline tests/cypress/support/step_definitions/I click on link.ts]: Should be in `jupyterlite`

Comment (VsevolodX) [reply tests/cypress/support/step_definitions/I click on link.ts]: A folder?

**Comment by VsevolodX** [comment]
Closing to split Docker related stuff and the task changes into two separate branches

**Pull request closed without merging**